docs/proposals/backup-restore.md

```diff
@@ -0,0 +1,260 @@
+
+#### Command: `oras restore`
+
+**Short summary:**
+Restore OCI artifacts or images from a local OCI image layout or archive into a registry.
+
+**Syntax:**
+```bash
+oras restore [flags] <registry>/<repository>[:<ref1>[,<ref2>...]] [...]
+```
+
+**Output:**
+Artifacts are uploaded to the target registry/registries as specified.
+
+**New flags:**
+
+- `--input <path>`: Required. Restore from from a folder or archive file to registry.
+- `--exclude-referrers`: Restore the image from backup without the referrers
+
+**Common flags:**
+
+* `--concurrency <int>`: Number of parallel upload operations. Default: `3`.
+* `--plain-http`: Allow insecure connections to registry without SSL check.
+* `--insecure`: Allow connections to registries without valid TLS certificates.
+* `--registry-config <path>`: Path to the authentication configuration file for the registry.
+* `--username <string>`: Username for authenticating to the registry.
+* `--password <string>`: Password for authenticating to the registry.
+* `--password-stdin`: Read password from stdin.
+* `--identity-token <string>`: Use bearer token for authentication.
+* `--identity-token-stdin`: Read identity token from stdin.
+* `--ca-file <path>`: Path to custom CA certificate file.
+* `--cert-file <path>`: Path to client TLS certificate file.
+* `--key-file <path>`: Path to client private key file.
+* `--resolve <host:port:address[:address_port]>`: Customized DNS for registry.
+* `--debug`: Output debug logs (implies `--no-tty`).
+* `--distribution-spec string`: [Preview] set OCI distribution spec version and API option for target. Options: v1.1-referrers-tag, v1.1-referrers-api
+* `--no-tty`: Disable progress bars.
+
+### User Experience in the CLI
+
+**Backup and Restore an Entire Repository and Tagged Artifacts**
+
+```bash
+# Backup a repository from a registry to a local compressed tarball
+oras backup --include-referrers registry.acme.com/repo:v1 --output multi-backup.tar
+
+# Transfer the backup file to new environment via secure channels (e.g., BitLocker-enabled removable drives)
+
+# Restore images and referrer artifacts from a local backup file to a target registry
+oras restore registry.backup.com/repo:v1  --input multi-backup.tar 
+```
+
+The backup includes images and all linked referrers. 
+
+**Backup and Restore Multiple Repositories**
+
+```bash
+# Backup multiple repositories from a registry to a local OCI image layout
+$ oras backup registry.k8s.io/kube-apiserver-arm64:v1.31.0 registry.k8s.io/kube-controller-manager-arm64:v1.31.0 --output k8s-control-plane
+
+# List the tags in the OCI image layout
+$ oras repo tags --oci-layout k8s-control-plane
+registry.k8s.io/kube-apiserver-arm64:v1.31.0
+registry.k8s.io/kube-controller-manager-arm64:v1.31.0
+
+# Restore them to two repositories in different registries
+$ oras restore localhost:5000/kube-apiserver-arm64:v1.31.0 localhost:5000/kube-controller-manager-arm64:v1.31.0 --input k8s-control-plane:v1.31.0
```

Comment:
The `restore` command example is ambiguous. You might need to re-think about it.